pkg/config/config.go

```diff
@@ -365,6 +370,10 @@ func (nsxConfig *NsxConfig) validate(enableVPC bool) error {
 	if err := nsxConfig.validateCert(); err != nil {
 		return err
 	}
+	if nsxConfig.LicenseValidationInterval == 0 {
```

Comment:
The logic is not correct for no DFW license case

Reply:
remove it